ribbon/src/main/java/feign/ribbon/LBClientFactory.java

```diff
@@ -21,10 +23,18 @@ public LBClient create(String clientName) {
     }
   }
 
+  IClientConfigKey<String> ServerThrottledStatusCodes = new CommonClientConfigKey<String>("ServerThrottledStatusCodes") {};
```

Comment:
Yes, though consistent with the similar statics in CommonClientConfigKey itself, so I thought their might be some weird convention about CommonClientConfigKey instances. If not, I'll change it to something more conventional.